### ISSUE TITLE
[Accessibility][JAWS] Unable to read and select options for multiselect dropdown

### ISSUE THREAD

**Issue by tpanpalia** [OfficeDev/office-ui-fabric-react#7469]
Issue : JAWS is unable to read and select the options for Office Fabric multiselect dropdown when user navigates using down arrow key (Chrome).

Expected Behavior : While using JAWS, user should be able to read the options and select the option on focus (Chrome).

**Comment by atneik** [comment]
@betrue-final-final

**Comment by dzearing** [comment]
Are there specific violations of aria specifications we're doing here? Is Narrator correct or is Jaws correct?

We should follow up with the Narrator team here. (and see if we can get a github contact we can ping here.)

**Comment by dzearing** [comment]
If we are following spec and Jaws can't read it we should not change anything here.

**Comment by smhigley** [comment]
Hi @atneik, how would you feel about me picking up this issue? I've been working on custom select accessibility patterns, and have a couple thoughts about how to fix it. I've checked the current implementation in NVDA/FF, Edge/Narrator, and Chrome/JAWS and can reproduce unexpected behavior in all of them, although the specific behavior varies. Chrome/JAWS certainly has the most unusable experience of all of them.

I'm going to run a couple tests of different implementations across browser/screen reader combinations, but I think it's likely the best fit will be a variant of the `role="combobox"` [pattern](https://www.w3.org/TR/wai-aria-practices/#combobox) with a readonly input. A button/listbox is also a possibility, but so far I've heard better things about `combobox` for select dropdowns.

Let me know how that sounds, and if you have any concerns or thoughts about how best to implement it. Otherwise, I'm looking forward to diving in!

**Comment by atneik** [comment]
@smhigley That would be great! I'll forward you the email thread during my investigation.

**Comment by atneik** [comment]
@smhigley are you able to assign yourself to this issue?

**Comment by smhigley** [comment]
@atneik it doesn't look like I can assign myself, but my github account should be linked to Microsoft already. If you can assign it to me, go ahead 👍
